src/test/java/uk/gov/justice/probation/courtcasematcher/messaging/MessageParserTest.java

```diff
@@ -58,14 +58,14 @@
 
         @Autowired
         @Qualifier("commonPlatformJsonParser")
-        public MessageParser<CommonPlatformHearing> messageParser;
+        public MessageParser<CPHearing> messageParser;
```

Comment:
Again probably not part of your change but HEARING_DATE and START_TIME constants in this class are unused